docs/content/ingestion/compaction.md

```diff
@@ -47,7 +48,8 @@ Compaction tasks merge all segments of the given interval. The syntax is:
 |`dataSource`|DataSource name to be compacted|Yes|
 |`interval`|Interval of segments to be compacted|Yes|
 |`dimensions`|Custom dimensionsSpec. compaction task will use this dimensionsSpec if exist instead of generating one. See below for more details.|No|
-|`keepSegmentGranularity`|If set to true, compactionTask will keep the time chunk boundaries and merge segments only if they fall into the same time chunk.|No (default = true)|
+|`segmentGranularity`|If this is set, compactionTask will change the segment granularity. `keepSegmentGranularity` must be set to false or null to use this option. If this is not set, the original segment granularity will be kept if keepSegmentGranularity is null or true. Otheriwse, `ALL` segment granularity will be used.|No|
```

Comment:
> Otheriwse, `ALL` segment granularity will be used.

Suggest clarifying this with some more detail on what the "ALL" granularity means

Might be helpful also to include the table from this comment: https://github.com/apache/incubator-druid/pull/6758#issuecomment-451004778

Reply:
Thanks. Added more docs.